.github/stale.yml

```
@@ -1,14 +1,23 @@
 # Number of days of inactivity before an issue becomes stale
 daysUntilStale: 30
+
 # Number of days of inactivity before a stale issue is closed
 daysUntilClose: 7
-any-of-issue-labels: unconfirmed,cant-reproduce,stale
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels: [cant-reproduce]
+
 # Label to use when marking an issue as stale
 staleLabel: stale
+
 # Comment to post when marking an issue as stale. Set to `false` to disable
 markComment: >
   This issue has been automatically marked as stale because it has not had
   recent activity. It will be closed if no further activity occurs. Thank you
   for your contributions.
```

Maybe it is beneficial to clarify the message as well to give the creator some more information?

```suggestion
markComment: >
  This issue has been automatically marked as stale because it could not be
  reproduced by the team, and has not had new input for some while. It will
  be closed if no further activity occurs.<br>
  Thank you for your contribution and bringing this to our attention though.
```

---

I think cleaner to leave as is, thanks though!